### ISSUE TITLE
Table Export broken

### ISSUE THREAD

**Issue by joyfullservice** [joyfullservice/msaccess-vcs-integration#3]
Function `GetTableExportSql` should set dbs object before referencing tabledef.

```
Set dbs = CurrentDb
Set tdf = dbs.TableDefs(strTable)
intFields = tdf.Fields.Count
```

**Comment by pmidhdev** [comment]
Hi, still new with GitHub put perhaps you want check my changes on branch LIBVCS, where both issue are patched, plus other changes

**Comment by A9G-Data-Droid** [comment]
Ok, I think I got it this time. I have a new pull that has no merge conflicts. I did not change any of the code from @pmidhdev.

**Comment by A9G-Data-Droid** [comment]
Starting over. The new PR #9 contains only bugfixes without any of the functional changes introduced by previous PRs.

**Comment by A9G-Data-Droid** [comment]
Closed by #9
